src/js/_enqueues/vendor/jquery/jquery.serialize-object.js

```diff
@@ -1,7 +1,7 @@
 /*!
- * jQuery serializeObject - v0.2 - 1/20/2010
+ * jQuery serializeObject - v0.2-wp - 1/20/2010
```

Comment:
Logic check: no update available and WP has needed to take over maintenance of these vendor libraries?

Reply:
Correct, library not maintained since 2010 (ref https://github.com/cowboy/jquery-misc/)